aws/provider.go

```diff
@@ -155,7 +155,9 @@ func Provider() terraform.ResourceProvider {
 			},
 		},
 
-		DataSourcesMap: map[string]*schema.Resource{},
+		DataSourcesMap: map[string]*schema.Resource{
+			"aws_caller_identity": dataSourceAwsCallerIdentity(),
```

Comment:
@beauknowssoftware I added this data source just to make it possible to get account number from within the code for other things we'll need to do and easier testing.

Reply:
Why would we need this?  The standard AWS provider should work for this, right?

Reply:
I agree, but there was no way to test the provider using a `terraform plan` or `terraform apply` without having at least something in the Terraform.  The other resources in the provider didn't make sense to create to test, I figured this was the simplest way of testing the provider to verify credentials and the workload without creating resources.  Open to other suggestions.